### PR TITLE
release 0.9

### PR DESCRIPTION
**Pull request by bayolau**
@marghoob @johnmu I'm making some last-minute changes to the build system. Don't have time to retest everything end-to-end. Please let me know if you have the slightest worry that anything would break.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3,12 +3,12 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.bina.lrsim</groupId>
-    <artifactId>lrsim</artifactId>
-    <version>0.1</version>
+    <artifactId>LongISLND</artifactId>
+    <version>0.9</version>
```
Comment (bayolau) [inline pom.xml]: @johnmu @marghoob this should change `target/lrsim-0.1.jar` to `target/LongISLND-0.9.jar`, would there be any side effects outside of the `target` directory?

Comment (johnmu) [inline pom.xml]: only if other people are using your artifact as a dependency...

File: pom.xml [hunk 2]
```diff
@@ -3,12 +3,12 @@
...
     <packaging>jar</packaging>
 
-    <name>lrsim</name>
-    <url>http://maven.apache.org</url>
+    <name>LongISLND</name>
+    <url>http://bioinform.github.io/longislnd/</url>
```
Comment (bayolau) [inline pom.xml]: @johnmu @marghoob these are proper way of filling these fields?

Comment (johnmu) [inline pom.xml]: That looks fine.

**Comment by bayolau** [comment]
Thanks @johnmu

**Pull request merged**